app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -102,19 +102,52 @@ def default_authentication_type
     :bearer
   end
 
-  def scan_job_create(entity_class, entity_id)
+  def scan_job_create(entity)
+    check_policy_prevent(:request_containerimage_scan, :raw_scan_job_create, entity)
```

Comment:
Hmm.  You're right about `Job.create_job()`.
But first `check_policy_prevent` calls `enforce_policy` calls `MiqEvent.raise_evm_event` with options containing entity.  Does `MiqEvent.raise_evm_event` queue things?
Let's see...  It calls `MiqAeEvent.raise_evm_event` which calls `call_automate` which may or may not queue:
https://github.com/ManageIQ/manageiq/blob/85f4150b480a47b6836f17e357cdf4bfb7cce430/lib/miq_automation_engine/engine/miq_ae_event.rb#L140-L145
it seems nothing on this chain set `options[:synchronous]` so it will queue, but I might be wrong...